cmd/oras/root/pull.go

```diff
@@ -159,6 +159,15 @@ func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget,
 		}
 	}
 
+	const (
+		Downloading = "Downloading"
+		Pulled      = "Pulled     "
+		Processing  = "Processing "
+		Skipped     = "Skipped    "
+		Restored    = "Restored   "
+		Downloaded  = "Downloaded "
```

Comment:
Do we need to export these?

Reply:
No, since different command contains different prompt and all prompts used by one command should be padding to the maximum length. E.g.  `Skipped` for `oras pull` should be `Skipped    `, but for `oras cp` should be `Skipped`